kevm-pyk/src/kontrol/__main__.py

```diff
@@ -426,6 +439,19 @@ def parse(s: str) -> list[T]:
     solc_to_k_args.add_argument('contract_file', type=file_path, help='Path to contract file.')
     solc_to_k_args.add_argument('contract_name', type=str, help='Name of contract to generate K helpers for.')
 
+    def _test_type(value: str) -> tuple[str, str | None]:
+        # Make sure that it matches the regex if the input is incomplete
+        if value.find(',') == -1:
```

Comment:
Could you insert this find in the regex directly?

Reply:
Right, that's a good idea, thank you!